src/components/ContentHighlights/HighlightStepper/ContentHighlightStepper.jsx

```diff
@@ -171,13 +262,13 @@ const ContentHighlightStepper = ({ enterpriseId }) => {
                 <Stepper.ActionRow.Spacer />
                 <Button
                   variant="tertiary"
-                  onClick={() => setCurrentStep(STEPPER_STEP_LABELS.CREATE_TITLE)}
+                  onClick={handleNavigateFromSelectContent}
```

Comment:
nit: would it make sense to name this function `handleNavigateToCreateTitle` vs. `handleNavigateFromSelectContent`?